src/librustc_middle/hir/mod.rs

```diff
@@ -78,8 +78,8 @@ pub fn provide(providers: &mut Providers<'_>) {
         let module = hir.as_local_hir_id(id.to_def_id()).unwrap();
         &tcx.untracked_crate.modules[&module]
     };
-    providers.hir_owner = |tcx, id| tcx.index_hir(id.krate).map[id.index].signature;
-    providers.hir_owner_items =
-        |tcx, id| tcx.index_hir(id.krate).map[id.index].with_bodies.as_ref().map(|items| &**items);
+    providers.hir_owner = |tcx, id| tcx.index_hir(LOCAL_CRATE).map[id].signature;
```

Comment:
did all this random stuff come from the rebase?

Reply:
Yeah, reviewing commit by commit will give some issues since I had to fix/revert some things like this in the last commit :(